templates/CRM/Contact/Page/DedupeFind.tpl

```diff
@@ -324,8 +324,7 @@
       var is_selected = CRM.$('.crm-dedupe-select-all').prop('checked') ? 1 : 0;
     }
 
-    var criteria = {/literal}'{$criteria|escape}'{literal};
```

Comment:
This escape wasn't working - hence the switch to cachekey string in this function